static/assets/constants.js

```diff
@@ -0,0 +1,17 @@
+/* eslint quotes: ["error", "double", "avoid-escape"] */
+/* eslint quote-props: [2, "always"]*/
+/* Don't modify anything outside the {} brackets.
+ * Insides of the {} brackets should be formatted as a JSON object.
+ * JSON rules:
+ * 1. All keys and string values must be enclosed in double quotes.
+ * 2. Each key/value pair should be on a new line.
+ * 3. All values and keys must be constant, you can't use any Javascript
+ *    functions.
+ */
+var constants = {
```

Comment:
Actually I think you can just declare the constants on oppiaFoundationWebsite directly?

In general assets/constants.js is only meant to be for constants that are shared between frontend and backend. Here, though, the backend doesn't use these.

Reply:
I thought constants.js was a neat idea, kind of a parity between config/feconf for backend and constants.js for frontend. I guess I will do it the normal way then.

Reply:
Done!